src/components/Charts/publications/editeurs/dynamique-ouverture/chart-taux-ouverture.js

```diff
@@ -58,13 +58,13 @@ const Chart = ({ hasFooter, hasComments, id, domain }) => {
     });
 
     Axios.post(ES_API_URL, query, HEADERS).then((response) => {
-      setPublishers(
-        response.data.aggregations.by_publisher.buckets
-          .filter((item) => item.key !== 'Cold Spring Harbor Laboratory')
-          .map((item) => ({ label: item.key, value: item.key })),
-      );
+      const opts = response.data.aggregations.by_publisher.buckets
+        .filter((item) => item.key !== 'Cold Spring Harbor Laboratory')
+        .map((item) => ({ label: item.key, value: item.key }));
+      opts.unshift({ label: capitalize(intl.formatMessage({ id: 'app.all-publishers' })), value: '*' });
```

Comment:
Maybe we could refactorize in one function this loop with filter() and unshift(), with an explicit name